hoot-js/src/main/cpp/hoot/js/schema/JsonOsmSchemaLoader.cpp

```diff
@@ -131,7 +131,7 @@ QStringList JsonOsmSchemaLoader::_asStringList(const QVariant& v) const
   if (v.type() == QVariant::List)
   {
     QVariantList l = v.toList();
-    result.reserve(l.size());
+    //result.reserve(l.size());
```

Comment:
Why comment this out? Either leave it or remove it. I don't think it does any harm, but doesn't help much either.